lib/alg/index.js

```diff
@@ -8,6 +9,7 @@ module.exports = {
   postorder: require("./postorder"),
   preorder: require("./preorder"),
   prim: require("./prim"),
+  shortestPaths: require("./shortest-paths"),
```

Comment:
It looks good that you extract it but the documentation will need to mention that this API can run bellman ford or Dijkstra